includes/admin/helper/class-wc-helper.php

```diff
@@ -1046,12 +1046,29 @@ public static function get_local_woo_plugins() {
 			$data['_product_id']      = absint( $product_id );
 			$data['_file_id']         = $file_id;
 			$data['_type']            = 'plugin';
+			$data['slug']             = dirname( $filename );
 			$woo_plugins[ $filename ] = $data;
 		}
 
 		return $woo_plugins;
 	}
 
+	/**
+	 * Obtain a list of slugs of locally installed Woo extensions.
+	 * (Cached for 1 day.)
+	 */
+	public static function get_local_woo_plugin_slugs() {
+		$woo_plugin_slugs = get_transient( 'wc_helper_installed_woo_plugin_slugs' );
```

Comment:
I'd actually cache get_local_woo_plugins() instead since that will cover future usage.

As for getting slugs, try `wp_list_pluck` - it's magic.

Reply:
Aha! I was googling for a pluck yesterday, but didn't find `wp_list_pluck` or `wc_list_pluck`!

Went with `wp_list_pluck` as we don't need the extra flexibility of the woo wrapper. 48946b3cc6d8e16da067ca35b27ff5d63efbae06